components/article/test/view.coffee

```diff
@@ -194,10 +192,17 @@ describe 'ArticleView', ->
 
     it 'on small screens, resets image sizes for imageset previews', ->
       @view.windowWidth = 600
-      @imgsFillContainer.callCount.should.be.above 1
+      @imgsFillContainer.callCount.should.equal 1
 
   describe '#imgsFillContainer', ->
     it 'returns true if images are narrower than their container', ->
       container = @view.$('.article-section-artworks ul').width(1400)
       imgsFillContainer = @view.imgsFillContainer([{width: 600}, {width:700}], container, 5)
       imgsFillContainer.isFilled.should.equal true
+
+  describe '#renderCalloutSections', ->
+
+    it 'renders callouts', ->
+      articles = [_.extend(fixtures.article, { thumbnail_title: 'callout article' })]
+      Backbone.sync.args[0][2].success results: articles
```

Comment:
The `Articles` collection [overrides](https://github.com/artsy/force/blob/master/collections/articles.coffee#L10) Backbone's parse, which is why the fixtures weren't getting converted to models correctly earlier! thought I was goin nuts.